ledger/error/src/test/suite/scala/com/daml/error/DamlContextualizedErrorLoggerSpec.scala

```diff
@@ -13,7 +13,8 @@ class DamlContextualizedErrorLoggerSpec extends AnyFlatSpec with Matchers {
   it should "sort entries by keys and skip empty values" in {
     val contextMap = Map("c" -> "C", "a" -> "A", "b" -> "B", "empty value" -> "")
 
-    val actual = ErrorCode.formatContextAsString(contextMap)
+    val actual =
+      DamlContextualizedErrorLogger.forTesting(getClass).formatContextAsString(contextMap)
```

Comment:
would be nice to align the spec name (`DamlContextualizedErrorLoggerSpec`) with the name of the class it is testing (i.e. `ContextualizedErrorLogger`), though I am not sure this method should belong then to that trait and be actually in `DamlContextualizedErrorLogger` object

Reply:
Ok, done